.travis.yml

```diff
@@ -26,8 +26,8 @@ deploy:
   - provider: script
     script: "npx lerna publish from-package --yes --dist-tag rc"
     on:
-      branch: production
-      tags: true
+      all_branches: production
+      tags: on
```

Comment:
I'm not sure about these changes. I don't know what our goal is, but the format seems to contradict Travis docs: https://docs.travis-ci.com/user/deployment#conditional-releases-with-on.
According to the docs, the only possible value for `all_branches` is `true` and `tags` can be either `true`, `false`, or a tag name.

Reply:
oh, sorry, this was not meant to be committed, I had been testing different things with the production builds. I will revert this before merging.